src/main/java/kitchenpos/application/MenuGroupService.java

```diff
@@ -1,26 +1,29 @@
 package kitchenpos.application;
 
-import kitchenpos.dao.MenuGroupDao;
-import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.menu.MenuGroupRepository;
+import kitchenpos.dto.menu.MenuGroupRequest;
+import kitchenpos.dto.menu.MenuGroupResponse;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 
 @Service
 public class MenuGroupService {
-    private final MenuGroupDao menuGroupDao;
 
-    public MenuGroupService(final MenuGroupDao menuGroupDao) {
-        this.menuGroupDao = menuGroupDao;
+    private final MenuGroupRepository menuGroupRepository;
+
+    public MenuGroupService(final MenuGroupRepository menuGroupDao) {
```

Comment:
```suggestion
    public MenuGroupService(final MenuGroupRepository menuGroupRepository) {
```

전체적으로 빈 주입을 위한 생성자에서 네이밍이 바뀌지 않은게 있는 것 같네요 :) 확인 부탁드려요~

Reply:
앗 감사합니다